onnx_mxnet/backend.py

```diff
@@ -74,15 +119,19 @@ def run_node(cls, node, inputs, device='CPU'):
         mod.bind(for_training=False, data_shapes=data_shapes, label_shapes=None)
 
         # initializing parameters for calculating result of each individual node
-        mod.init_params()
+        if int(len(params)) > 0:
+            mod.set_params(arg_params=params, aux_params=None)
```

Comment:
you might want to put `arg_params=params, aux_params=params`
for some operators having aux_params like "BatchNorm"